examples/with-nx/apps/app/index.d.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+declare module '*.svg' {
+  const content: any;
+  export const ReactComponent: any;
```

Comment:
Should this type export `ReactComponent` if `svgr: false` is the default in `next.config.js`?

Reply:
Yea I took a closer look here and just removed this completely - another remnant of the boilerplate.